src/rules/indentRule.ts

```diff
@@ -57,16 +77,32 @@ export class Rule extends Lint.Rules.AbstractRule {
 class IndentWalker extends Lint.RuleWalker {
     private failureString: string;
     private regExp: RegExp;
+    private size: number;
+    private replaceRegExp: RegExp;
+    private replaceIndent: string;
 
     constructor(sourceFile: ts.SourceFile, options: Lint.IOptions) {
         super(sourceFile, options);
 
+        if (this.hasOption(OPTION_INDENT_SIZE_2)) {
+            this.size = OPTION_INDENT_SIZE_2;
+        } else if (this.hasOption(OPTION_INDENT_SIZE_4)) {
+            this.size = OPTION_INDENT_SIZE_4;
+        } else {
+            this.size = OPTION_INDENT_SIZE_DEFAULT;
```

Comment:
how do I tell the rule not to mess with *my* indentation level if it defaults to `4`?
I'd suggest not to fix anything if there is no indentSize specified.